### ISSUE TITLE
Email Ticket

### ISSUE THREAD

**Issue by silicahd** [nextcloud/mail#9353]
### Is your feature request related to a problem? Please describe.

Lets rethink the mail box as it stands. We get a lot of requests and get buried by 10-20 emails per day. We can mark a flag on each one but going back to resolve it gets lost in all the new emails. 90% of emails for businesses are requests so why not group as ticket. 

### Describe the solution you'd like

Let say that you get a request in an email. It would be nice to have a button that marks the email as a "ticket" this now lives at the top of the the emails or maybe in another folder called tickets where you can get a new view of all the open tickets or maybe ability to add new statuses. Any new email in that thread will go there. You can not have the ability to let say add comments or collaborate with others in the company on this via a comment to the left of the screen. Maybe the ability to add collaborators from the company directory. Now this will also show in there ticket view. Once a ticket is complete you can mark it as complete. Any new email will not go into imbox like all others unless it is marked again as ticket. 

### Describe alternatives you've considered

Well no company out there does this. I have been able to use system like Zoho Desk but these request to add a new email address and it is a completely new system. There is no reason for any business to have two systems for such a feature.

**Comment by marius-wieschollek-valantic** [comment]
What you're describing is essentially a service desk / issue tracking system, not a mail client.
E-Mail and mail clients like this app are not designed for this task. It will always be labor intensive to get even somewhat organized, as mails don't fulfill the usual requirements of these systems. (e.g. issue numbers, issue report requirements, issue status, time tracking, change tracking, assigning people to it …)
Once all your requests are sorted into a "tickets" folder, you're just going to end up with a full "tickets" folder where everything gets lost.

The described functionality is far outside what a mail client should be doing . How do you imagine comments on an email? Everyone else would have to have access to that mail, otherwise they can't comment on it.

I'd recommend you look for a service desk / help desk / project management application that meets your needs. You try and see if you can adapt your workflow to use apps like [tasks](https://apps.nextcloud.com/apps/tasks) or [deck](https://apps.nextcloud.com/apps/deck). Or look at the integrations for dedicated solutions such as [OpenProject](https://apps.nextcloud.com/apps/integration_openproject), [Jira](https://apps.nextcloud.com/apps/integration_jira) or [Zammad](https://apps.nextcloud.com/apps/integration_zammad).

**Comment by alexanderdd** [comment]
@ChristophWurst ok to close as "not planned"?
